ontology/docs/hvac_chws.md

```diff
@@ -0,0 +1,329 @@
+# Chilled & Condensing Water Systems (CHWS & CWS)
+
+## Type Definition
+Hydronic cooling and condensing systems. Chilled water systems cool water mechanically, which circulates to end-use devices; they dump the heat absorbed from the chilled water loop either to atmosphere ('air-cooled') or to another water distribution system ('water-cooled'). Condensing systems serve those mechanical devices which rely on water-source heat rejection.
+
+There are many varieties of chilled & condenser water system, but some common examples include:
+- **Constant volume** (predominantly on condensing water systems), where pumps run at constant speeds providing fixed flowrates at given stages.
+- **Variable primary**, where there is one loop between the chillers and the end devices. The pumps operate at variable speed.
+- **Primary/Secondary**, where there are two separate loops, with the primary serving as the production loop and the secondary serving as the distribution loop.
+
+## Type Requirements
+- Chillers require the mechanical cooling of water.
+- Condensers require that, regardless of their own cooling method (sensible or evaporative), they serve downstream devices which require water as a condensing source.
+- Heat exchangers provide heat transfer between two fluid streams, without intermixing the fluids. They are modeled separately only if they are stand-alone devices. 
+
+## Example: Variable Primary CHWS (Air-Cooled) 
+This version of CHWS has a single set of variable speed pumps which serve air-cooled chillers and downstream equipment.
+
+### BMS Example
+![CHWS Variable Primary](./figures/bms_screenshots/chw_aircooled.png)
+**Notes:**
+- The fields associated with individual devices (individual chiller commands, for example) get mapped to those devices; system level telemetry gets assigned to the system directly.
+
+### System Diagram and Connections
+![CHWS Variable Primary](./figures/system_diagrams/chws_aircooled.png)
+
+### Sample Building Config
+```yaml
+BLDG-1:
+  type: FACILITIES/BUILDING
+
+ZONE-1:
+  connections:
+    BLDG-1: CONTAINS
+  type: FACILITIES/ZONE
+
+ZONE-2:
+  connections:
+    BLDG-1: CONTAINS
+  type: FACILITIES/ZONE
+
+CHWSYS-BLDG-1:
+  connections:
+    BLDG-1: CONTAINS
+    FCU-1: FEEDS
```

Comment:
I think this is backwards. Per the [docs](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md#relationships),     

> # Listed entities are sources on connections
 so FCU-1 should have a connection CHWSYS-BLDG-1: FEEDS
and AHU-1 should have a connection CHWSYS-BLDG-1: FEEDS

because the CHWSYS feeds the AHU and FCU not the other way around.

Reply:
adjusted.